integrations/tekton/README.md

```diff
@@ -0,0 +1,28 @@
+# Tekton Pipeline Example
+
+This example pipeline allows you to easily run an ORT scan on [Tekton](https://tekton.dev/), some knowledge about Tekton will help to understand this example pipeline.. This example pipeline works best on [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine).
+
+## Getting Started
+To run the pipeline, you will need the following things:
+* A Kubernetes Cluster with Tekton installed (follow the [installation instructions](https://tekton.dev/docs/pipelines/install/))
+* You will need to install the [git-clone](https://hub.tekton.dev/tekton/task/git-clone) and [gcs-upload](https://hub.tekton.dev/tekton/task/gcs-upload) tasks
+* A [Google Cloud Storage](https://cloud.google.com/products/storage) bucket
+
+The pipeline definition is in the `ort-pipeline.yaml` you will need to replace `ort:latest` with the full image name e.g. `gcr.io/<project_id>/ort:latest`.
+
+The pipeline takes a few inputs:
+* `shared-data` is a workspace and will need a volume, it's used for checking out the source code repo, and storing the temporary artifacts
+* `gcs-credentials` is a workspace and will need a volume. On GKE you can use Workload Identities to allow the KSA to access the GCS bucket for uploading the results. (If you are not using Workload Idenity, find more information how to configure credentials in the [gcs-upload](https://hub.tekton.dev/tekton/task/gcs-upload) documentation)
+* `repo-url` is a param, and takes the path to the Git repo that should be scanned
```

Comment:
s/param/parameter/

(Unless "param" really is a coined term here. Same below.)